api/controllers/userController.js

```diff
@@ -0,0 +1,29 @@
+const {userService} = require('../services')
+const { asyncWrap } = require('../utils/error')
+
+const signUp =asyncWrap(async (req,res) => {
+    const {email, password, phoneNumber, name} = req.body;
+
+    if(!email || !password || !phoneNumber || !name) {
+        const error = new Error('KEY_ERROR')
+        error.statusCode = 400;
+
+        throw error
+    }
+    await userService.signUp(email, password, phoneNumber, name)
+
+    res.status(200).json({message:"success"});
```

Comment:
```suggestion
    res.status(201).json({message:"success"});
```
다음에는 데이터가 생성됨을 알리는 `201 Created` Status Code를 활용해 보세요!

Reply:
확인 후 직접 하단에 `Merge pull request` 눌러서 머지해보세요!